### PR TITLE
Fix deprecated size prop usage in Loading component

### PR DESCRIPTION
**Pull request by tmasuyama1114**
**Description:**

This pull request addresses issue [#2488](https://github.com/yamada-ui/yamada-ui/issues/2488), which pertains to the documentation of the `Loading` component. The documentation has been updated to replace references to the deprecated `size` prop with the correct `fontSize` prop to align with the current API standards of the `@yamada-ui/react` package.

## Current behavior (updates)

The documentation for the `Loading` component was referencing the deprecated `size` prop, which is no longer supported and has been replaced with the `fontSize` prop in the latest API.

## New behavior

The `Loading` component's documentation now correctly reflects the use of the `fontSize` prop instead of the deprecated `size` prop. This ensures that users following the documentation will use the component correctly.

## Is this a breaking change (Yes/No):

No

## Additional Information

- This update is purely a documentation change. No functional changes were made to the component or its associated tests.
- The documentation is now consistent with the current API, reducing potential confusion for developers.

### CODE REVIEW

**Review by quantumshiro — DISMISSED** [review]

**Comment by hirotomoyamada** [comment]
@tmasuyama1114 @quantumshiro 

Hi, thanks for the PR.

Isn't the issue you are specifying a documentation update?
It appears that you are changing the package instead.

**Comment by tmasuyama1114** [comment]
@hirotomoyamada 
Hi, thanks for pointing that out!

I see now that the issue was intended to be a documentation update only.
My apologies for misunderstanding; I initially thought the URL in the issue was just for reference 🙏 

I've reverted the changes to the package and updated the documentation as needed.
Could you please review the PR again?

**Review by taroj1205 — APPROVED** [review]
LGTM Thanks!

File: docs/contents/components/feedback/loading/theming.ja.mdx
Comment (hirotomoyamada) [inline docs/contents/components/feedback/loading/theming.ja.mdx:1]: This file is an automatically generated file and should not be modified.

File: docs/contents/components/feedback/loading/props.mdx
Comment (hirotomoyamada) [inline docs/contents/components/feedback/loading/props.mdx:1]: This file is an automatically generated file and should not be modified.

File: docs/contents/components/feedback/loading/props.ja.mdx
Comment (hirotomoyamada) [inline docs/contents/components/feedback/loading/props.ja.mdx:1]: This file is an automatically generated file and should not be modified.

File: docs/contents/components/feedback/loading/theming.mdx
Comment (hirotomoyamada) [inline docs/contents/components/feedback/loading/theming.mdx:1]: This file is an automatically generated file and should not be modified.

**Comment by tmasuyama1114** [comment]
@hirotomoyamada 
I see that the files I modified are automatically generated and should not be manually edited.
Thanks for pointing this out, and I have updated the PR accordingly.

**Review by taroj1205 — APPROVED** [review]
LGTM Thanks!

**Comment by hirotomoyamada** [comment]
Thanks

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**